package.json

```diff
@@ -38,6 +38,7 @@
   "author": "GoDaddy.com Operating Company LLC",
   "license": "MIT",
   "dependencies": {
+    "@jsdevtools/readdir-enhanced": "^6.0.3",
```

Comment:
note: used this new dependency for converting to async/await  - it is MIT licensed[ here](https://www.npmjs.com/package/readdir-enhanced)

Reply:
@Swaagie /@indexzero assuming ^ is ok?

Reply:
Seems fine to me, do we buy him a 🌴 ? 😁